foundation_cms/campaigns/models/campaign_page.py

```diff
@@ -0,0 +1,198 @@
+from django.db import models
+from django.shortcuts import redirect, render
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
+from wagtail.fields import RichTextField
+from wagtail.models import Page
+from wagtail_localize.fields import SynchronizedField, TranslatableField
+
+from foundation_cms.base.models import AbstractBasePage
+from foundation_cms.nothing_personal.models.article_page import (
+    NothingPersonalArticlePage,
+)
+from foundation_cms.utils import get_default_locale, localize_queryset
+
+from .cta_base import CTA
+from .petition import Petition
+
+
+class CampaignPage(AbstractBasePage):
+    """
+    These pages come with sign-a-petition CTAs
+    """
+
+    header = models.CharField(max_length=250, blank=True, help_text="Header for the campaign page")
+
+    cta = models.ForeignKey(
+        CTA,
+        related_name="campaign_page_for_cta",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text="Choose one of our call-to-action snippets, or create a new one.",
+    )
+
+    # --- SIGNED STATE FIELDS ---
+    signed_header = models.CharField(
+        max_length=200, default="Thank you for signing", help_text="Header shown after petition is signed"
+    )
+
+    signed_donation_question = models.CharField(
+        max_length=200,
+        blank=True,
+        default="Would you like to support our work with a donation?",
+        help_text="Donation question shown after petition is signed",
+    )
+
+    signed_body = RichTextField(
+        default="Thanks so much for signing this petition asking "
+        "for transparency on data used to train AI tool! Can you "
+        "add a donation to support Mozilla's  non-profit work for trustworthy AI?",
+        help_text="Content shown after signing",
+    )
```

Comment:
Hi @adalcima I think that this thank you message was specific to a legacy campaign that was related to AI. Can we check with stakeholders what the default thank you message should be?